modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -1191,9 +1269,15 @@ operator<<(std::ostream & out, const FeatureFloodCount::FeatureData & feature)
 {
   static const bool debug = true;
```

Comment:
It's a local member purely for debugging and since it's const it doesn't really matter that it's static.